packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_webview_controller.dart

```diff
@@ -259,6 +259,12 @@ abstract class PlatformWebViewController extends PlatformInterface {
     throw UnimplementedError(
         'setUserAgent is not implemented on the current platform');
   }
+
+  /// Toggle the listener for scroll position
+  Future<void> setScrollListener(void Function(int x, int y)? onOffsetChange) {
```

Comment:
To follow the current convention of the interface, this should probably be `setOnScrollChanged(...)` or `setOnContentOffsetChanged(...)`. I'm impartial to either of these.